lomond/message.py

```diff
@@ -106,7 +106,7 @@ def from_payload(cls, payload):
             text = payload.decode('utf-8')
         except UnicodeDecodeError as error:
             raise errors.CriticalProtocolError(
-                'payload contains invalid utf-8 ({})',
+                'payload contains invalid utf-8; {}',
```

Comment:
In practice, this probably can't occur, since the utf8 has been validated. It's more of a sanity check now.